src/test/groovy/.#GithubWorkflowRunTests.groovy

```diff
@@ -0,0 +1 @@
+user@lws7.140641
```

Comment:
What's this?

Reply:
it's lock file created by editor. Removed this file.